roles/tftp-server/defaults/main.yml

```diff
@@ -9,6 +9,14 @@ apt_proxy: false
 # Defaults to deb.debian.org | archive.ubuntu.com
 #mirror: deb.debian.org
 
+web_root: /var/www/html
+
 user_name: videoteam
-user_password: null
-pw_salt: U1ESiTXUQE0tFh2r4LRl
+# how to make the hash:
+# $ printf "r00tme" | mkpasswd -s -m md5
+user_password_hash: $1$G1uRZoLB$r68rPXmpJl/C6ervlynKa/
```

Comment:
The default doesn't belong here, it should probably default to null here (and have an appropriate if block in the preseed template)